nextflow.config

```
@@ -44,6 +51,10 @@ params {
   variant_snp_filter = false
   outdir = './results'
 
+  validate_params = true
+  show_hidden_params = false
+  schema_ignore_params = 'genomes,input_paths'
```

If not using `--genome`

```suggestion
  schema_ignore_params = 'input_paths'
```

---

As above, any reason that you need to keep this?